shipyard_blueprints/s3/shipyard_s3/cli/authtest.py

```diff
@@ -1,31 +1,26 @@
 import os
 import sys
-import boto3
 from shipyard_s3 import S3Client
+from shipyard_templates import ShipyardLogger
+from dotenv import load_dotenv, find_dotenv
+
+load_dotenv(find_dotenv())
+
+logger = ShipyardLogger.get_logger()
 
 
 def main():
-    s3 = S3Client(
-        aws_access_key=os.getenv("AWS_ACCESS_KEY_ID"),
-        aws_secret_access_key=os.getenv("AWS_SECRET_ACCESS_KEY"),
-    )  # only necessary for logging
     try:
-        client = boto3.client(
-            "sts",
-            aws_access_key_id=os.getenv("AWS_ACCESS_KEY_ID"),
+        s3 = S3Client(
```

Comment:
may be worth writing tests for the variations of connect. Mainly thinking of the optional region field. The previous method was not attempting to return the client but return the caller's identity.